admin-manual/maintenance/cli-import-export.rst

```diff
@@ -1282,11 +1284,13 @@ can be found in the User Manual, here:
    * :ref:`csv-import-authority-records-gui`
    * :ref:`csv-import-authority-record-relationships-gui`
 
-Run the CSV import task from AtoM's root directory. To use the example authority record import file that is included with the AtoM installation:
+Run the CSV import task from AtoM's root directory. To use the example authority
+record import file that is included with the AtoM installation:
 
 .. code-block:: bash
 
-   php symfony csv:authority-import lib/task/import/example/authority_records/example_authority_records.csv --index
+   php symfony csv:authority-import lib/task/import/example/authority_records/
+   example_authority_records.csv --index
```

Comment:
i'd say don't worry about fixing this, but FYI command-line blocks are sometimes the one place where i make an exception on the 80-char rule, just in case when the formatting is rendered on the website, it comes out strange and people get confused when trying to implement the command.

Reply:
Yeah, I thought I had reverted all the code-block lines but missed this one. Fixed it now.